README.md

```diff
@@ -23,33 +23,35 @@ npm install
-| LOW_PRIORITY_TOPIC_ARN            | test                                                  | Doesn't need to be set to a real ARN unless your work touches it specifically                                                              |
-| MIXPANEL_TOKEN                    | none                                                  | Found in Mixpanel at Settings > Project Settings > Project Token                                                                           |
-| ARCHIVEMATICA_BASE_URL            | none                                                  | Found in EC2, not needed unless you're running the cleanup cron                                                                            |
-| ARCHIVEMATICA_API_KEY             | none                                                  | Found in Bitwarden, not needed unless you're running the cleanup cron                                                                      |
+| MAILCHIMP_COMMUNITY_LIST_ID       | 2736f796db                                            | The default value corresponds to the `dev` list                                                                     |
+| SENTRY_DSN                        | none                                                  | Can be found in Sentry under Projects > stela > Settings > Client Keys (DSN)                                        |
+| DEV_NAME                          | none                                                  | Only set in local environments. Should be your given name. all lowercase. Used to create Sentry envs for developers |
+| AWS_REGION                        | us-west-2                                             |                                                                                                                     |
+| AWS_ACCESS_KEY_ID                 | none                                                  | The same one you use in `devenv`                                                                                    |
+| AWS_SECRET_ACCESS_KEY             | none                                                  | The same one you use in `devenv`                                                                                    |
+| LOW_PRIORITY_TOPIC_ARN            | test                                                  | Doesn't need to be set to a real ARN unless your work touches it specifically                                       |
+| MIXPANEL_TOKEN                    | none                                                  | Found in Mixpanel at Settings > Project Settings > Project Token                                                    |
+| ARCHIVEMATICA_BASE_URL            | <none. required for running tests>                    | It is the url of the EC2 instance on which archivematica is running (a machine named dev-archivematica)             |
```

Comment:
I ask because I think it's better not to call out to a real Archivematica if possible. The fix here might be to make changes to the tests so that they run without a base url for now, until we decide what should happen to the local environment as we move to Archivematica.

Reply:
on a second look, it seems that the thing blocking tests from running with missing env vars is the requireEnv function. they seem to run fine with a fictional value, not an emprty string.

Reply:
Will you include that a fake value will work, to save the next person some time looking up the actual url?